sidechain/storage/src/storage_tests_get_blocks_following.rs

```diff
@@ -0,0 +1,132 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{error::Error, test_db_fixture::TestDbFixture};
+use itp_types::ShardIdentifier;
+use its_primitives::{
+	traits::SignedBlock,
+	types::{BlockHash, SignedBlock as SignedSidechainBlock},
+};
+use its_test::sidechain_block_builder::SidechainBlockBuilder;
+use sp_core::{ed25519, Pair, H256};
+use std::{
+	assert_matches::assert_matches,
+	time::{SystemTime, UNIX_EPOCH},
+};
+
```

Comment:
Tests for the new storage method `get_blocks_following()`.
(uses the new `TestDbFixture` struct)